kb/plane_geometry/formal_statements/statements/statements_scs/stat_triangle_be_inscribed_side_radius_perimeter.scs

```diff
@@ -0,0 +1,14 @@
+stat_triangle_be_inscribed_side_radius_perimeter <- statement;;
+
+stat_triangle_be_inscribed_side_radius_perimeter -> rrel_key_sc_element: concept_triangle; nrel_be_inscribed; nrel_radius; nrel_perimeter;;
+
+stat_triangle_be_inscribed_side_radius_perimeter => nrel_main_idtf: [Акс.(треугольник;быть вписанным*;радиус*;периметр*)] (*<- lang_ru;;*);;
+stat_triangle_be_inscribed_side_radius_perimeter => nrel_main_idtf: [Ax.(triangle;be inscribed*;radius*;perimeter*)] (*<- lang_en;;*);;

```

Comment:
It isn't an axiom. It is a statement. You should use [Stat.(...)] insted of [Ax.(...)]